src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Color color = ColorSupplier.getRandomColor();
+        int number = new Random().nextInt(100);
+        return new Ball(color,number);
+    }
+

```

Comment:
redundant empty line
```suggestion
```

Reply:
if I delete this line, the checkstyle plugin says that it is an error 
![idea64_uOrCPhgxNl](https://github.com/mate-academy/jv-lottery/assets/153805566/4efbde55-495b-438a-87b8-b3cbd7e4d7af)

Reply:
Everything is fine now, just wanted to say that the thing here is that Github shows it as a single line but in fact there were 2, you can invest some time and check how these 3 cases would look in the IDE and on github:
* file ending without a new line
* file ending a sible new line
* file ending with 2 empty lines